ModelModel/UsersCell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UsersTableViewCell.swift
+//  ModelModel
+//
+//  Created by Stanislav Cherkasov on 13.12.17.
+//  Copyright © 2017 Stanislav Cherkasov. All rights reserved.
+//
+
+import UIKit
+
+class UsersCell: UITableViewCell {
+  
+    var userRow: String
+  
+        init(userRow: String) {
```

Comment:
```
 initWith(user: User) {
     self.user = user
}
```

Find a place where you should make all UI initialization e.g.
self.nameLabel.text = self.user.name